.goreleaser.yml

```
@@ -143,16 +149,33 @@ brews:
       system "#{bin}/regen --version"
     install: |
       bin.install "regen"
+      bash_completion.install "completions/regen.bash" => "regen"
+      zsh_completion.install "completions/regen.zsh" => "_regen"
+      fish_completion.install "completions/regen.fish"  
 
 nfpms:
-  - maintainer: regen-network
+  - 
+    maintainer: regen-network
```

Any reason for changing the format here?

```suggestion
  - maintainer: regen-network
```